core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -869,38 +867,11 @@ maybe_start_metaflows(Call, Endpoints, _CallId) ->
     of
         'false' -> 'ok';
         'undefined' ->
-            [maybe_start_metaflow(Call, Endpoint) || Endpoint <- Endpoints],
+            [konami_util:maybe_start_metaflow(Call, Endpoint) || Endpoint <- Endpoints],
```

Comment:
No core libraries can add a dependency on a Kazoo application, as that would require the app be installed alongside core.

Reply:
Understand